### PR TITLE
Added the code to toonify any image

### LINKED ISSUE #562: Toonify Image
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**
This is a python application that takes an image as input and toonify it (make a cartoonist image of it) using https://deepai.org/machine-learning-model/toonify API.

**Describe alternatives you've considered**

**Additional context**

@HarshCasper may I work on it

### PR DESCRIPTION
**Pull request by Tejas1510**
# Description

Please include a summary of the change and which issue is fixed. List any dependencies that are required for this change.

Fixes #562

Replace `issue_no` with the issue number which is fixed in this PR

## Type of change

_Please delete options that are not relevant._

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update
- [x] Documentation Update

# Checklist:

- [x] My code follows the style guidelines(Clean Code) of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have created a helpful and easy to understand `README.md`
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests/screenshots(if any) that prove my fix is effective or that my feature works

### CODE REVIEW

**Comment by Tejas1510** [comment]
@HarshCasper can you please review the pull request

File: Python/ToonifyImage/toonify.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import requests
+import shutil # to save image locally
+imagePath=input("Please enter the path of image to toonify : ")
+api_key=input("Please enter your api key")
```
Comment (HarshCasper) [inline Python/ToonifyImage/toonify.py:4]: Please have the API Key in a `.env` file and use a module like dotenv to import them in your Python Code

File: Python/ToonifyImage/toonify.py [hunk 2]
```diff
@@ -0,0 +1,26 @@
+import requests
+import shutil # to save image locally
+imagePath=input("Please enter the path of image to toonify : ")
```
Comment (HarshCasper) [inline Python/ToonifyImage/toonify.py:3]: Can you try passing it via arguments instead of dynamic input?

**Review by HarshCasper — CHANGES_REQUESTED** [review]
The Code looks good to me! 

Please make the below changes and it would be ready to merge!

**Comment by Tejas1510** [comment]
@HarshCasper I have made changes taking input as an argument instead of dynamic input, all the comments are added above the code, Just the part of API in env I don't have an idea about it.
Rest I have done
Please Give it a look

**Comment by HarshCasper** [comment]
Hello @Tejas1510 

That's easy. Let me explain: 

- Have a `.env` file in your Directory and add the following: 

```
API_KEY = "XXXXXXXXXXXXXXXXXXXXXXX"
```

- Install a Package named `decouple` to get the Environment Variables. Import it like this: 

`
from decouple import config
`

- Go to your Main Script and add the following line of code: 

`
api_key = config("API_KEY")
`

You are now ready to use the API Key in your Code. Do not push the `.env` in your Code because its not a good practise. Have a `.env.example` which the User can rename to `.env` and store their API Key. Have the `.env.example` for sure.

**Comment by Tejas1510** [comment]
Hey @HarshCasper thanks for explaining the concept of .env file in a very simpler way
I have added the .env.example file as asked by you so the user can store their API here and use it main code
I have also updated the main file by the using  CONFIG
Please review if everything is all right

**Comment by HarshCasper** [comment]
LGTM!

**Comment by Tejas1510** [comment]
@HarshCasper  Can you please update the leaderboard with the point associated with merged PR

**Pull request merged**